### PR TITLE
fix: Unified card enrollment ios bug fixes

### PR DESCRIPTION
**Pull request by OmkarJ13**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=xAzvNNt)
### <samp>🤖 Generated by Copilot at 6876a33</samp>

Improved the UI and layout of the manage corporate cards and add corporate card pages. Fixed some HTML errors and footer overlap issues.

### <samp>🤖 Generated by Copilot at 6876a33</samp>

> _Oh we're the coders of the sea, and we work on the UI_
> _We fix the footers and the fonts, to make them look so nice_
> _We use `ion-footer` and `ion-toolbar`, and we check the HTML_
> _And we heave, and we ho, and we pull the code, on the count of three_

### <samp>🤖 Generated by Copilot at 6876a33</samp>

*  Replace the custom footer div with an `ion-footer` and an `ion-toolbar` to fix the overlapping issue and make the footer consistent with other pages ([link](https://github.com/fylein/fyle-mobile-app/pull/2345/files?diff=unified&w=0#diff-4e011a42cc83184b53a781fc09dfbe3eed19d4830bffc7fde2e27757934cf4b6L166-R179), [link](https://github.com/fylein/fyle-mobile-app/pull/2345/files?diff=unified&w=0#diff-bfcd5ac8d936b76f67c888c3ab4d32d8f299658abe49190d2cf8a1bfd22e3bc4L143-R146), [link](https://github.com/fylein/fyle-mobile-app/pull/2345/files?diff=unified&w=0#diff-a9990626fe6e417ac0088e9e79ea18f1e0b64e1f644e8cead61870540da85a5eL48-R48))
* Increase the font size of the card number and expiry labels to 14px to improve readability and match the design ([link](https://github.com/fylein/fyle-mobile-app/pull/2345/files?diff=unified&w=0#diff-bfcd5ac8d936b76f67c888c3ab4d32d8f299658abe49190d2cf8a1bfd22e3bc4R105), [link](https://github.com/fylein/fyle-mobile-app/pull/2345/files?diff=unified&w=0#diff-bfcd5ac8d936b76f67c888c3ab4d32d8f299658abe49190d2cf8a1bfd22e3bc4R118))
* Add closing tags for the `ion-toolbar` and the `fy-footer-cta-container` div to match the opening tags ([link](https://github.com/fylein/fyle-mobile-app/pull/2345/files?diff=unified&w=0#diff-a9990626fe6e417ac0088e9e79ea18f1e0b64e1f644e8cead61870540da85a5eL55-R55))

## Clickup
https://app.clickup.com

### CODE REVIEW

File: src/app/fyle/dashboard/dashboard.page.html [hunk 1]
```diff
@@ -1,62 +1,60 @@
-<div class="dashboard">
+<div class="dashboard--background-container">
```
Comment (OmkarJ13) [inline src/app/fyle/dashboard/dashboard.page.html:1]: Changed approach of showing background, previous approach was causing issues in ios

File: src/app/fyle/manage-corporate-cards/add-corporate-card/add-corporate-card.component.scss [hunk 2]
```diff
@@ -102,6 +102,7 @@
     align-items: center;
     justify-content: space-between;
     gap: 8px;
+    font-size: 14px;
```
Comment (Dimple16) [inline src/app/fyle/manage-corporate-cards/add-corporate-card/add-corporate-card.component.scss:105]: isn't `font-size: 14px` by default?

File: src/app/shared/components/add-card/add-card.component.scss [hunk 3]
```diff
@@ -12,6 +12,7 @@
   border: 1px dashed $grey-light;
   border-radius: 16px;
   background-color: $translucent-white-2;
+  font-size: 14px;
```
Comment (Dimple16) [inline src/app/shared/components/add-card/add-card.component.scss:15]: please verify this once if it's already there by default

**Review by Dimple16 — APPROVED** [review]
minor comments

File: src/app/fyle/manage-corporate-cards/add-corporate-card/add-corporate-card.component.scss (hunk 2 above)
Comment (OmkarJ13) [reply src/app/fyle/manage-corporate-cards/add-corporate-card/add-corporate-card.component.scss:105]: Yeah, I also thought the same, but checked this and looks like default is 16px, hence added this.
Ref: https://www.w3schools.com/css/css_font_size.asp#:~:text=1em%20is%20equal%20to%20the,size%20of%201em%20is%2016px.

**Review by Chethan-Fyle — APPROVED** [review]

**Pull request merged**